src/briefcase/platforms/linux/system.py

```diff
@@ -830,27 +848,25 @@ class LinuxSystemPackageCommand(LinuxSystemMixin, PackageCommand):
     def packaging_formats(self):
         return ["deb", "rpm", "pkg", "system"]
 
-    def _verify_deb_tools(self):
-        """Verify that the local environment contains the debian packaging tools."""
-        if not Path("/usr/bin/dpkg-deb").exists():
-            raise BriefcaseCommandError(
-                "Can't find the dpkg tools. Try running `sudo apt install dpkg-dev`."
-            )
-
-    def _verify_rpm_tools(self):
-        """Verify that the local environment contains the redhat packaging tools."""
-        if not Path("/usr/bin/rpmbuild").exists():
-            raise BriefcaseCommandError(
-                "Can't find the rpm-build tools. Try running `sudo dnf install rpm-build`."
-            )
+    def _verify_packaging_tools(self, app: AppConfig):
+        """Verify that the local environment contains the packaging tools."""
+        tool_name, executable_name, package_name = {
+            "deb": ("dpkg", "dpkg-deb", "dpkg-dev"),
+            "rpm": ("rpm-build", "rpmbuild", "rpmbuild"),
+            "pkg": ("makepkg", "makepkg", "pacman"),
+        }[app.packaging_format]
 
-    def _verify_pkg_tools(self):
-        """Verify that the local environment contains the arch packaging tools(ABS)."""
-        if not Path("/usr/bin/makepkg").exists():
-            raise BriefcaseCommandError(
-                "Can't find the `makepkg` tool. Try running `sudo pacman -Syu pacman`."
-                # makepkg is part of pacman package
-            )
+        if not self.tools.shutil.which(executable_name):
```

Comment:
I realized I couldn't attempt to use a system verifier here to check if the packages are installed...or at least not reliably.

The general problem is we know which package type we want to create....but we don't have the full matrix of packagers to distros; for instance, trying to build a RPM on Debian, running `sudo apt install rpmbuild` isn't going to get you the RPM packager....it's going to error.

So, I left all this as just trying to find the executable and dropping an 80/20 error message; most times, the error message will be correct....but it can definitely be wrong for mismatched packaging types to distro combinations.

Reply:
Yeah - this seems an entirely acceptable compromise. I'd argue its going to be a lot better than 80/20 - more like 99/1, because it's only people on *seriously* independent distros that are going to hit the missing case.